mlir/include/mlir/Dialect/Rock/Tuning/Serializable.h

```diff
@@ -60,6 +61,15 @@ template <class Derived, char Seperator = ','> struct Serializable {
 
   bool deserialize(const std::string &s) {
     auto out = static_cast<const Derived &>(*this);
+
+    const auto numCommas =
```

Comment:
Let's not do this hack.

Instead, could we, for example, make the perf config start with `v2:`?

If we see a perf config that doesn't start with `v2:`, we know it's an old perf config that doesn't have the split-K factor (so we can transparently add 1).

Reply:
Ok. Let me try it.

Reply:
Hi @krzysz00 

What do you think about the following?

https://github.com/ROCm/rocMLIR/blob/f78a327f85979cd6f37599abe5d5c6f598acaede/mlir/include/mlir/Dialect/Rock/Tuning/Serializable.h#L77-L94

This implementation supports the old and new formats. It can be additionally extended if needed